conftest.py

```diff
@@ -0,0 +1,151 @@
+import builtins
+from contextlib import contextmanager
+from pathlib import Path
+import yaml
+from typing import Callable
+
+import pytest
+
+
+# Prevent errors caused by pytest trying to collect tests from externals 
+# (e.g. from errors whilst importing python code in the UCLA-roms repo, see https://github.com/CWorthy-ocean/C-Star/issues/53)
+collect_ignore_glob = [
+    "cstar/cstar_local_config.py",
+    "cstar/externals/**"
+]
+
+
+EXAMPLE_BLUEPRINTS = {
```

Comment:
I feel like this list of example blueprints should live somewhere other than in `conftest.py`.